notebooks/en/enterprise_cookbook_dev_spaces.md

```diff
@@ -0,0 +1,88 @@
+
+Similar to services like Google Colab, you can change the hardware the Space is running on-the-fly. We recommend doing initial setup work on the upgraded or free CPU, for example data cleaning, setting up Endpoints, or testing APIs. Once your code is set up, you can simply click on `Settings` at the top right of the Space and change to a wide selection of hardware that might be required for more compute intensive inference or training jobs. When you change hardware, the Space will restart itself and all environment variables will be lost (like with Google Colab) and you will have a new clean environment on the new hardware after some seconds. Your stored and saved files (code, data etc.) will of course also be available on the new hardware. The image below shows the available hardware at the time of writing (June 2024) and this will be updated in the future. 
+
+In the bottom left of the image, you can also see the `Sleep time settings` where you can define how long you want the hardware to run in case of inactivity. This is a major advantage over Google Colab: if you want to save money, you can make the Space sleep after 15 minutes of inactivity, but if you need the hardware to be available for a 48 hour training run or longer, you can just prevent the Space from falling asleep and let it run for as long as you want. You can also manually `Pause` the Space and you will no longer be charged for the Space Hardware. 
+
+<img src="https://cdn-lfs-us-1.huggingface.co/repos/13/3d/133d8ca2460bf82ba2bdbe928d91a6c780364a6d0cf9005087db081cca492c02/ce651a2be35a88878755f41531c877b50ba194fce0971397010e2271904bbf39?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27enterprise-jupyterlab-hardware-options.png%3B+filename%3D%22enterprise-jupyterlab-hardware-options.png%22%3B&response-content-type=image%2Fpng&Expires=1717920105&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTcxNzkyMDEwNX19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy11cy0xLmh1Z2dpbmdmYWNlLmNvL3JlcG9zLzEzLzNkLzEzM2Q4Y2EyNDYwYmY4MmJhMmJkYmU5MjhkOTFhNmM3ODAzNjRhNmQwY2Y5MDA1MDg3ZGIwODFjY2E0OTJjMDIvY2U2NTFhMmJlMzVhODg4Nzg3NTVmNDE1MzFjODc3YjUwYmExOTRmY2UwOTcxMzk3MDEwZTIyNzE5MDRiYmYzOT9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoifV19&Signature=Ye2vpEAbFaQuVTru4iLEQncnF4VzON9mAEmkJqjyJQy9J0S4kA%7EtO3O3LwWFVHOT2esYGDjG1re0lwYLG3mft1lxieoJbm5vuwIPNELMSa-Vb3e7bA6ZuOcisklEoTwpRi0aQcKa6NlWoF3rNDFLwmLdZVQNHvFKCee6%7E0wiRol-znACCk-7uIoJ7dclyRmVmUgdsgskVfSVIXDYVZH%7EQyA5yH4AhNIwQhb5gNjhVZq0OG%7EbmCGkdszhLVD5ljw4E6Lage5kr1ID8owZFNI62jKbNHvIJS-MM0WtN3iu0zCt9mv28Py0A-UKDrBmbSvJQuCwvcRAlhjiLUPyApct2g__&Key-Pair-Id=KCD77M1F0VK2B">
+
+If you scroll down in the settings, you will see additional options, like expanding storage, resetting the space etc. In case you have not set a password during Space creation, you can also create a secret called `JUPYTER_TOKEN` here later, which will replace the default "huggingface" password.
```

Comment:
```suggestion
If you scroll down in the settings, you will see additional options, like expanding storage, resetting the Space, etc. In case you have not set a password during Space creation, you can also create a secret called `JUPYTER_TOKEN` here later, which will replace the default "huggingface" password.
```